ansible_collections/arista/avd/roles/eos_designs/doc/platform-settings.md

```diff
@@ -44,7 +58,7 @@ The reload delay values should be reviewed and tuned to the specific environment
 ## Default Values
 
 ```yaml
-platform_settings:
+default_platform_settings:
```

Comment:
please change this back to `platform_settings`

Reply:
Fixed in most recent commit.